tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+import logging
+
+import pexpect
+import pytest
+from assertpy import assert_that
+
+
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["awsbatch", "slurm"])
+def test_pcluster_configure(vpc_stack, key_name, region, os, instance, scheduler, clusters_factory, test_datadir):
```

Comment:
this test is not compatible with custom_ami and custom_url options, shall we skip it with an appropriate message in case such options are used with the tests rather than failing?

Reply:
Good point. I found six that looked like they should result in skips. Let me know if you disagree.